src/components/Header.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+export enum UpdatingStatus {
+  inProgres = 'inProgres',
+  success = 'success',
+  failed = 'failed',
+}
+
+type Props = {
+  onAdd: (value: string) => void;
+  status: UpdatingStatus;
+};
+
+export const Header: React.FC<Props> = ({ onAdd, status }) => {
+  const [title, setTitle] = useState('');
+  const titleField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    titleField.current?.focus();
+    if (status === UpdatingStatus.success) {
```

Comment:
you don't use `failed` status. So it can be replaced with a boolean value.
Moreover, it can be `const isInProgess = !!processingIds.length`

Variable name is only example